tests/unit/sidecars/live_data_watcher_loader_tests.py

```diff
@@ -0,0 +1,104 @@
+import os
+from unittest import mock
+import pytest
+import json
+
+import boto3
+from botocore.client import ClientError
+from moto import mock_s3
+
+from baseplate.sidecars.live_data_watcher import NodeWatcher
+from baseplate.sidecars.live_data_watcher import _load_from_s3
+from baseplate.sidecars.live_data_watcher import _parse_loader_type
+from baseplate.sidecars.live_data_watcher import LoaderType
+
+@pytest.fixture()
+def start_mock_s3():
+    with mock_s3():
+        yield
+
+@pytest.mark.parametrize('data,return_value', (
+    # Non-RAW configs
+    (json.dumps({'live_data_watcher_load_type': 'S3'}).encode('utf-8'), LoaderType.S3),
+
+    # Everything else should be RAW
+    (b'', LoaderType.RAW),
+    (json.dumps({'key': 'value'}).encode('utf-8'), LoaderType.RAW),
+    (json.dumps([]).encode('utf-8'), LoaderType.RAW),
+    (json.dumps(1234).encode('utf-8'), LoaderType.RAW),
+    (json.dumps(None).encode('utf-8'), LoaderType.RAW),
```

Comment:
What about the positive case? What if the contents don't parse as JSON at all?

Reply:
The positive test case is at the top. These are for negative parses, where it defaults back to RAW. Keep in mind that the data in S3 is not parsed, it's just dumped to a file.

Reply:
This is all testing parsing the znode data anyway. Positive case is at the top, still needs a 'non json' test case though. I'll add